api/pkg/service/admin/admin_http_test.go

```diff
@@ -46,6 +46,12 @@ const agentToken007Updated = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9." +
 	"eyJpZCI6MzEsIm5hbWUiOiJhZ2VudC0wMDEiLCJzY29wZXMiOlsidGVzdDpyZWFkIiwiYWdlbnQ6Y3JlYXRlIl0sInR5cGUiOiJhZ2VudCJ9." +
 	"0u_SkfkJjuq8jyax8jwLdAzHUl7J0g0a6jkN9gLoJl4"
 
+// Token for the user with github name "foi-boi" and github login "foi"
+// It has a extra scope "config:refresh" along with default scope
+const validTokenWithConfigRefreshScope = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9." +
+	"eyJpZCI6MTMsImxvZ2luIjoiZm9pIiwibmFtZSI6ImZvaS1ib2kiLCJzY29wZXMiOlsicmF0aW5nOnJlYWQiLCJyYXRpbmc6d3JpdGUiLCJjb25maWc6cmVmcmVzaCJdfQ." +
+	"7eA0J6xbOOkth2dsPd9z_VxmZVHjIwXamQWyCq-Jur8"
```

Comment:
can we add a utility to create jwt with scopes on demand?

Reply:
yeah will address in seperate pr for all tokens